### PR TITLE
lpc2list: remove deprecated functions

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @Skeen
Compile tested: ath79

### CODE REVIEW

**Comment by Skeen** [comment]
LGTM

File: devel/lpc21isp/patches/120-nanosleep.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+--- a/lpc21isp.c
++++ b/lpc21isp.c
+@@ -1130,7 +1130,8 @@ void ClearSerialPortBuffers(ISP_ENVIRONMENT *IspEnvironment)
+ */
+ void Sleep(unsigned long MilliSeconds)
+ {
+-    usleep(MilliSeconds*1000); //convert to microseconds
++    struct timespec m = {MilliSeconds * 1000 * 1000, 0};
```
Comment (Skeen) [inline devel/lpc21isp/patches/120-nanosleep.patch:8]: I thought the format here was `{seconds, nanoseconds}`?

Comment (neheb) [reply devel/lpc21isp/patches/120-nanosleep.patch]: Fixed.

**Comment by neheb** [comment]
Fixed.

**Pull request merged**